src/_guides/language/language-tour_migrated.md

```diff
@@ -2690,9 +2687,14 @@ class Point {
 All uninitialized instance variables have the value `null`.
 
 All instance variables generate an implicit *getter* method. Non-final
-instance variables also generate an implicit *setter* method. For details,
+and `late final` instance variables also generate
+an implicit *setter* method. For details,
```

Comment:
A `late final` field *with an initializer* does not generate an implicit setter. (I didn't realize this until last week.) So:

```dart
class Foo {
  late final x = 123;
}

main() {
  Foo().x = 234; // compile-time error, no setter.
}
```